src/sentry/api/event_search.py

```diff
@@ -121,7 +121,11 @@ class SearchKey(namedtuple('SearchKey', 'name')):
 
     @property
     def snuba_name(self):
-        return FIELD_LOOKUP[self.name]['snuba_name']
+        field = FIELD_LOOKUP.get(self.name)
+        if field:
+            return field['snuba_name']
+        # assume custom tag if not listed
+        return 'tags[%s]' % (self.name,)
```

Comment:
Can the tag name from the user contain `[` or `]`?

Reply:
no, but realized they can contain `:`, so going to say they have to be in quotes if they contain that (at least as a temp solution)

https://github.com/getsentry/sentry/blob/master/src/sentry/tagstore/base.py#L18